public/js/event-app.js

```diff
@@ -182,8 +182,33 @@ $(document).ready(function() {
         logger.log("Initialized app.");
 
         $("#admin-page-for-event").attr("href", "/admin/event/" + curEvent.id);
+        
+        var participantIndex = 1; 
+
+        $("#superuser-page-for-followupemail").attr("href", "/followup/event/" 
+            + curEvent.id + "/participant_" + participantIndex);
+        
         $(".about-event-container").hide();
 
+        //Show contact form on event page if either 
+        //preferred contact is null or noShare is false 
+
+        if(preferredContact == false && (noShare == false || noShare == undefined)) {
```

Comment:
Changes addressing the comment above 
https://github.com/drewww/unhangout/commit/f90e0bd749964cee71640183ef791fee86696f6f